src/App.js

```diff
@@ -1,25 +1,27 @@
 import React from 'react';
 import logo from './logo.svg';
 import './App.css';
+import {
+  BrowserRouter as Router, 
+  Switch, 
+  Route, 
+  Link
+} from "react-router-dom"; 
+import Nav from "./Nav";
+import Dashboard from "./Dashboard";
+import Organization from "./Organization";
+import NotFound from "./NotFound"
 
 function App() {
   return (
-    <div className="App">
-      <header className="App-header">
-        <img src={logo} className="App-logo" alt="logo" />
-        <p>
-          Edit <code>src/App.js</code> and save to reload.
-        </p>
-        <a
-          className="App-link"
-          href="https://reactjs.org"
-          target="_blank"
-          rel="noopener noreferrer"
-        >
-          Learn React
-        </a>
-      </header>
-    </div>
+    <Router>
+      <Nav />
+      <Switch>
+        <Route path="/organizations/:org" component={Organization}></Route>
```

Comment:
You should be able to close this with just /> since it doesn't actually contain anything